Example/Sources/Playgrounds/Avatar.playground/Contents.swift

```diff
@@ -6,7 +6,11 @@ import PlaygroundSupport
 //Get an image
 let testImage = #imageLiteral(resourceName: "NiceSelfi.jpg")
 
-var avatarView = AvatarView()
+var avatarView = AvatarView(frame: CGRect(x: 0, y: 0, width: 30, height: 30))
```

Comment:
So this was intended to be a demonstration of how little setup there was to use this feature. I would like the Avatar to be able to default to usable values by just calling `var avatarView = AvatarView()` and it just working for simplicity. But if that is still possible would you mind just adding it as a case in this file. I wanted this playground to be a place where people can experiment and see the capabilities of the library.

Reply:
But of coarse what you have replaced it with here should also be valid.

Reply:
@MacMeDan so when someone implement the sizing delegate, it creates image twice. If I leave it as it was, each time user want to create an avatar, it will be created first for `30x30` then the size user ask for. Thats why I implemented this way, but its up to you guys.